resources/sass/app.scss

```diff
@@ -190,3 +190,22 @@ body {
 .nav > a.nav-link.active:focus {
     color: rgba(0,0,0,.9);
 }
+
+#notificationDropdownMenu {
+    height: auto;
+    max-height: 400px;
+    overflow-x: hidden;
+}
+
+#markAllRead {
+    text-align: center;
+    background: none!important;
+  border: none;
+  padding: 0!important;
+  /*optional*/
+  font-family: arial, sans-serif;
```

Comment:
Why is this given a different font? I feel like the element this CSS is for would be better as a regular button. I'll have a play around tomorrow.